src/util/kaldi-thread.h

```diff
@@ -209,6 +209,9 @@ class TaskSequencer {
   void Wait() { // You call this at the end if it's more convenient
     // than waiting for the destructor.  It waits for all tasks to finish.
     if (thread_list_ != NULL) {
+      while (!thread_list_->thread.joinable()) {
+	sleep(1);
```

Comment:
can you please indent this 2 spaces?  also below.

Reply:
done. Sorry about the delay.